api/types.proto

```diff
@@ -696,3 +696,9 @@ message ManagerStatus {
 	// Reachability specifies whether this node is reachable.
 	RaftMemberStatus.Reachability reachability = 4;
 }
+
+message Device {
+	string source = 1;
+	string target = 2;
+	string permissions = 3;
```

Comment:
@stevvooe Sorry I've been busy. In this case, what should be the next steps? Do you need more information about how we use GPUs? Let me know what you want to do or what you need.